redbot/cogs/mutes/mutes.py

```diff
@@ -0,0 +1,1212 @@
+            success = await self.channel_unmute_user(
+                channel.guild, channel, author, member, _("Automatic unmute")
+            )
+            if success["success"]:
+                try:
+                    await modlog.create_case(
+                        self.bot,
+                        channel.guild,
+                        datetime.now(timezone.utc),
+                        "cunmute",
+                        member,
+                        author,
+                        _("Automatic unmute"),
+                        until=None,
+                    )
+                except RuntimeError as e:
+                    log.error(_("Error creating modlog case"), exc_info=e)
+        except discord.errors.Forbidden:
+            return
+
+    @commands.Cog.listener()
+    async def on_member_update(self, before: discord.Member, after: discord.Member):
+        """
+        Used to handle the cache if a member manually has the muted role removed
+        """
+        guild = before.guild
+        mute_role_id = await self.config.guild(before.guild).mute_role()
+        mute_role = guild.get_role(mute_role_id)
+        if not mute_role:
+            return
+        b = set(before.roles)
+        a = set(after.roles)
+        roles_removed = list(b - a)
+        roles_added = list(a - b)
+        if mute_role in roles_removed:
+            # send modlog case for unmute and remove from cache
+            if guild.id not in self._server_mutes:
+                # they weren't a tracked mute so we can return early
+                return
+            if after.id in self._server_mutes[guild.id]:
+                try:
+                    await modlog.create_case(
+                        self.bot,
+                        guild,
+                        datetime.utcnow(),
+                        "sunmute",
+                        after,
+                        None,
+                        _("Manually removed mute role"),
+                    )
+                except RuntimeError as e:
+                    log.error(_("Error creating modlog case"), exc_info=e)
+                del self._server_mutes[guild.id][after.id]
+        if mute_role in roles_added:
+            # send modlog case for mute and add to cache
+            if guild.id not in self._server_mutes:
+                # initialize the guild in the cache to prevent keyerrors
+                self._server_mutes[guild.id] = {}
+            if after.id not in self._server_mutes[guild.id]:
+                try:
+                    await modlog.create_case(
+                        self.bot,
+                        guild,
+                        datetime.utcnow(),
+                        "smute",
+                        after,
+                        None,
+                        _("Manually applied mute role"),
+                    )
+                except RuntimeError as e:
+                    log.error(_("Error creating modlog case"), exc_info=e)
```

Comment:
Logging should not be translated.
```suggestion
                    log.error("Error creating modlog case", exc_info=e)
```